src/deprecated_utils.py

```diff
@@ -115,3 +116,19 @@ def extract_installer(release_image, dest):
             f"oc adm release extract --registry-config '{pull_secret}'"
             f" --command=openshift-install --to={dest} {release_image}"
         )
+
+
+def extract_version(release_image):
```

Comment:
Sounds weird to add it to a module named "deprecated_utils"

Reply:
It's almost identical to the `extract_installer` that's already used by the bootstrap test anyway, so it made sense to put it right next to it, even though the file is being deprecated.

Do you have any suggestions of where else it would be better to put it?

Reply:
@eliorerz can you advice?

Reply:
We probably missed `extract_installer` when we moved bootstrap_in_place to the new workflow.
I think that both `extract_installer` and `extract_version` should be removed from `deprecated_utils.py` module.
If this utils are used only by `test_bootstrap_in_place.py` I think  that we can create a dedicated utils file (under utils package) for bootstrap_in_place, another option is placing it on `utils.py`